modules/margin/src/test/resources/lch-trades.txt

```diff
@@ -0,0 +1 @@
+CobDate	LchMatchedTradeRef	TradeDate	LchRegistrationDateTime	MaturityDate	Currency	NotionalAmount	MatchingService	ExternalSourceRef	ProductType	CompoundingTrade	Party_A_SdMnemonic	Party_A_MbrMnemonic	Party_A_Account	Party_A_SdTradeId	Party_A_LegType	Party_A_Rate	Party_A_Spread	Party_A_IndexName	Party_A_FirstTenor	Party_A_DayCountFraction	Party_A_BusinessDayConvention	Party_A_EffectiveDate	Party_A_PmtFrequency	Party_A_RollDate	Party_A_FrontStubEndDate	Party_A_BackStubStartDate	Party_B_SdMnemonic	Party_B_MbrMnemonic	Party_B_Account	Party_B_SdTradeId	Party_B_LegType	Party_B_Rate	Party_B_Spread	Party_B_IndexName	Party_B_FirstTenor	Party_B_DayCountFraction	Party_B_BusinessDayConvention	Party_B_EffectiveDate	Party_B_ClientShortName	Party_B_PmtFrequency	Party_B_RollDate	Party_B_FrontStubEndDate	Party_B_BackStubStartDate	TradeLevelNPV	TradePV01	NettingEligible	EID	LID	NID	StandardNettingKey	BlendedNettingKey	InfTradePV01	Party_A_InflInterpMethod	Party_A_InflLag	Party_A_InflIndexLevelInitial	Party_A_InflIndexLevelFinal	Party_B_InflInterpMethod	Party_B_InflLag	Party_B_InflIndexLevelInitial	Party_B_InflIndexLevelFinal	XCCYPV01	XCCYDiscountingCurr
01/07/2016	LCH00000000001	01/07/2016	01/07/2016 17:10	05/07/2018	NOK	1495000000	SWAPSWIRE	1-2	IRS	N	XXXXNO22XXX	DBS	H	44778419FO	FLT	0		NIBOR-NIBR	6M	ACT/360	MODFOLLOWING	05/07/2016	6M	5							FIX	0.0087				30/360	MODFOLLOWING	05/07/2016		1Y	5			0	-222637.36	N			
01/07/2016	LCH00000000002	01/07/2016	01/07/2016 15:36	06/07/2026	USD	10000000	SWAPSWIRE	2-2	IRS	N	XXXXDKKKXXX	DBS	H	44778562C	FIX	0.0034				30/360	MODFOLLOWING	05/07/2016	1Y	5							FLT	0		EURIBOR-Reuters	6M	ACT/360	MODFOLLOWING	05/07/2016		6M	5			-7188.05	9511.04	N			
01/07/2016	LCH00000000003	01/07/2016	01/07/2016 08:06	06/07/2026	EUR	25000000	SWAPSWIRE	3-2	IRS	N	XXXXDKKKXXX	DBS	H	44765892C	FIX	0.003545				30/360	MODFOLLOWING	05/07/2016	1Y	5							FLT	0		EURIBOR-Reuters	6M	ACT/360	MODFOLLOWING	05/07/2016		6M	5			-54669.7	23797.81	N			
01/07/2016	LCH00000000004	01/07/2016	01/07/2016 08:47	05/07/2056	EUR	6000000	SWAPSWIRE	4-2	IRS	N	XXXXDKKKXXX	DBS	H	44766695C	FLT	0		EURIBOR-Reuters	6M	ACT/360	MODFOLLOWING	05/07/2016	6M	5							FIX	0.007938				30/360	MODFOLLOWING	05/07/2016		1Y	5			49315.56	-21155.36	N			

```

Comment:
This diff makes me wonder if the file was copied correctly. LCH files are notoriously difficult to copy segments from, because of the weird escape codes. Perhaps it is just that each \M should be a \N ?